### ISSUE TITLE
EPIC - Navigate with expression

### ISSUE THREAD

**Issue by tuliopereirazup** [ZupIT/beagle#860]
## Description
Currently the Navigate actions does not support expression on Route.Remote url, and this could add extra possibilities to Beagle Navigation.

Adding expressions to Route.Remote url will also add a problem to pre-fetch logic, because pre-fetch is execute before any expression is evaluated, consequently if a url has expression and should pre-fetch we decided to ignore this pre-fetch.

TLDR: Add expression support to Navigate Route.Remote url and ignore pre-fetch if the url has expression.

## Related issues

- [x] Android:  
- [x] iOS: #911 
- [x] Web: https://github.com/ZupIT/beagle-web-core/issues/232
